odoo/models.py

```diff
@@ -2781,11 +2781,11 @@ def qualify(field):
             if forbidden:
                 _logger.info(
                     _('The requested operation cannot be completed due to record rules: Document type: %s, Operation: %s, Records: %s, User: %s') % \
-                    (self._name, 'read', ','.join([str(r.id) for r in self][:6]), self._uid))
+                    (self._name, 'read', ','.join([str(r.id) for r in forbidden]), self._uid))
```

Comment:
why removing the `[:6]` here?

Reply:
The message seems to indicate that it shows all ids. 

My second concern that this forbidden list show the records that where excluded by the application of all the access rules and having all of them in one log statement give help to see the big picture. This avoid to go through forbidden ids 6 by 6 to fix an access rule issues.